--- ./db/models/fields/related.py	2013-08-13 21:05:59.418270268 +0100
+++ ./db/models/fields/related.py	2013-08-13 21:05:46.298269910 +0100
@@ -309,8 +309,9 @@
 
         related_pk = getattr(instance, self.related.field.rel.get_related_field().attname)
         if related_pk is None:
-            raise ValueError('Cannot assign "%r": "%s" instance isn\'t saved in the database.' %
-                                (value, instance._meta.object_name))
+            pass
+            #raise ValueError('Cannot assign "%r": "%s" instance isn\'t saved in the database.' %
+                                #(value, instance._meta.object_name))
 
         # Set the value of the related field to the value of the related object's related field
         setattr(value, self.related.field.attname, related_pk)
